src/ignitus-Dashboard/ignitus-DashboardHome/ignitus-DashboardHomeHeader/Components/index.tsx

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
Starting from the header, the implemented header has many spacing issues. eg. the position of ignitus icon is on the very left, Space between Home and internships, search bar does not have the icon, etc.
<img width="1440" alt="Screenshot 2020-06-10 at 9 44 07 AM" src="https://user-images.githubusercontent.com/26546816/84240946-0d9d0c80-aaff-11ea-8344-0c87750ece65.png">

<img width="738" alt="Screenshot 2020-06-10 at 9 46 34 AM" src="https://user-images.githubusercontent.com/26546816/84241106-4ccb5d80-aaff-11ea-99f8-133a726db680.png">

Reply:
Also, there should be a background-color so that the cards don't overlap with the header. 

<img width="1440" alt="Screenshot 2020-06-10 at 9 49 51 AM" src="https://user-images.githubusercontent.com/26546816/84241496-d7ac5800-aaff-11ea-93af-cdc383ed6fb6.png">